### DISCUSSION TITLE
Deleting Mod Deleted my Configuration Files

### DISCUSSION THREAD

**Discussion by luxmanifestus** [X2CommunityCore/xcom2-launcher#302] [General]
I'd like to give some feedback. I deleted a mod with the AML that had been updated and needed to be redownloaded to load correctly. In the process, my custom configuration files present in the mod directory to make items buildable in Prototype Proving Ground were also deleted. It would be better if files added to the directory remained after deletion of the mod rather than the entire directory being wiped out. Please modify the delete mod functionality so that any custom files are retained, as is the case when one unsubscribes to a mod through the workshop. Thanks.

**Comment by RevZero** [comment]
Yes, we discussed this in the past. The idea is to have a proper way to manage the configs which potentially includes being able to restore changes, even after a mod has been deleted and was re-installed. #53
